### ISSUE TITLE
add support for rendering other pages

### ISSUE THREAD

**Issue by notslang** [carrot/roots-tumblr#7]
these additional pages include:
- pages for older posts: `/page/<page number>`
- single post pages: `/post/<id>/<optional title>`, where the title doesn't matter, but is the page that gets redirected to from every other page that is missing the correct title.
- the rss feed: `/rss`
- user-based submissions: `/submit`
- random page (just links to a single-post-page): `/random`
- `/ask`
- custom pages defined in the `Pages` block

**Comment by kylemac** [comment]
great call

**Comment by hhsnopek** [comment]
For the custom pages it would be cool to have an object to define url paths that are different or make more sense for the actual page. That way we could have a folder that contains all the `content` which then holds all the content including pages: `content/customPage1`, `content/path/customPage2`, etc

```
custom:
    "customAwesomePathThatsNotWhatIWant/**": "doge/**"
    "blog/**": "DaBlog/**"
```

**Comment by notslang** [comment]
@hhsnopek: yeah, that's what https://github.com/jenius/roots/issues/527 is about too. The idea of a 1:many relationship is a huge thing that roots is missing right now :(

**Comment by hhsnopek** [comment]
Can't this be local server rewrites during watch and then on compile put it into the correct folder based on the specified path?

**Comment by notslang** [comment]
no... it needs to actually render all the files, based on some specification of what data goes to each template
